docs/en/sql-reference/statements/create/table.md

```diff
@@ -620,11 +641,57 @@ SELECT * FROM base.t1;
 ┌─n─┐
 │ 3 │
 └───┘
+```  
+</TabItem>
+<TabItem value="cloud_replace_example" label="Cloud">
+
+Consider the following table on ClickHouse Cloud: 
+
+```sql
+CREATE DATABASE base;
+CREATE OR REPLACE TABLE base.t1 (n UInt64, s String)
+ENGINE = MergeTree
+ORDER BY n;
+INSERT INTO base.t1 VALUES (1, 'test');
+SELECT * FROM base.t1;
+```
+
+```text
```

Comment:
`nit`: You can include the result in the same `sql` snippet above (also applicable to other places). Also add a line spacing between each SQL statement (referring to my other comment above).

Reply:
Agreed, easier to read with the result in the same snippet.